src/encoded/static/components/filegallery.js

```diff
@@ -2192,7 +2192,10 @@ const AnalysesSelector = ({ analyses, selectedAnalysesIndex, handleAnalysesSelec
             <div className="analyses-selector analyses-selector--file-gallery-facets">
                 <h4>Choose analysis</h4>
                 <select className="analyses-selector" value={selectedAnalysesIndex} onChange={handleSelection}>
-                    {analyses.map((analysis, index) => (
+                    // In ENCODE releases, release year is not in the data. To get around this, if you strip out the text and 
```

Comment:
Good comment, but it doesn‘t show up as a comment because it’s embedded in JSX instead of Javascript. You either need to convert it to {/* comment */} or move it inside the analyses.sort loop. It also has a couple trailing spaces at the ends of the lines.

Reply:
I did it the way I did because comments add download time to HTML pages. But if comments are needed in downloaded HTML file, that is doable.